docs/rfcs/UserProfilesv2_schema.json

```diff
@@ -0,0 +1,486 @@
+            "default": false,
+            "examples": [
+              true
+            ]
+          },
+          "primary": {
+            "$id": "/properties/identities/items/properties/primary",
+            "type": "boolean",
+            "title": "The Primary Schema ",
+            "description": "If true, indicates that this is the primary identity for the user, and thus the `identity.email` field also matches `primaryEmail`.",
+            "default": false,
+            "examples": [
+              true
+            ]
+          },
+          "lastModified": {
+            "$id": "/properties/identities/items/properties/lastModified",
+            "type": "string",
+            "title": "The Lastmodified Schema ",
+            "description": "Date of last modification of this identity. If the date is recent and `identity.verified` is set to true, this provides greater assurance that the identity currently belong to the user.",
+            "default": "",
+            "examples": [
+              "2017-03-09T21:28:51.851Z"
+            ]
+          },
+          "verifier": {
+            "$id": "/properties/identities/items/properties/verifier",
+            "type": "string",
+            "title": "The Verifier Schema ",
+            "description": "A unique name representing the verifier utilized in order to set this identity as verified. If set to `user-supplied`, then `identity.verified` is necessarily set to false",
+            "default": "",
+            "examples": [
+              "ad|Mozilla LDAP", "github", "google-oauth2", "passwordless", "user-supplied"
+            ]
+          },
+          "user_id": {
+            "$id": "/properties/identities/items/properties/user_id",
+            "type": "string",
+            "title": "The User_id Schema ",
+            "description": "The unique and stable user identifier for this identity. Note that `identities.user_id` may be different from the top-level `user_id` as it's associated with a specific identity. Do not use this value in order to identify the user profile in Mozilla IAM. This value is useful in order to query the original identity provider. When no specific `user_id` is supplied by the identity provider, `identities.user_id` is equal to `identities.email`",
+            "default": "",
+            "examples": [
+              "dn=mozilla.com,cn=lmcardle", "092394", "lmcardle"
+            ]
+          }
+        },
+        "required": [
+          "email",
+          "verified",
+          "primary",
+          "lastModified",
+          "verifier",
+          "user_id"
+        ]
+      }
+    },
+    "phoneNumbers": {
+      "$id": "/properties/phoneNumbers",
+      "type": "array",
+      "items": {
+        "$id": "/properties/phoneNumbers/items",
+        "type": "string",
+        "title": "The 0th Schema ",
+        "description": "All phone numbers that are associated with the user profile. Order is undefined.",
```

Comment:
Are these verified?

Reply:
Just saying, that's a good thing to document :)

Reply:
Currently we have no verified phone numbers though its technically possible, there's no plan to implement verification. We could have a generic optional attribute though.

Reply:
My point is mostly that it would be good to document that the phone number is not verified.

Just in case someone decides to use an API to write text messages or build a call robot :)

It's more of a good practice kind of thing... In reality people probably won't expect it to be validated.

Reply:
✓